tests/other_test.go

```diff
@@ -0,0 +1,93 @@
+			List: []string{
+				"module.ocp_base_cluster_1.null_resource.confirm_network_healthy",
+				"module.ocp_base_cluster_1.null_resource.reset_api_key",
+				"module.ocp_base_cluster_2.null_resource.confirm_network_healthy",
+				"module.ocp_base_cluster_2.null_resource.reset_api_key",
+			},
+		},
+		IgnoreUpdates: testhelper.Exemptions{ // Ignore for consistency check
+			List: []string{
+				"module.observability_agents_1.helm_release.logdna_agent",
+				"module.observability_agents_1.helm_release.sysdig_agent",
+				"module.observability_agents_2.helm_release.logdna_agent",
+				"module.observability_agents_2.helm_release.sysdig_agent",
+			},
+		},
+		ImplicitDestroy: []string{ // Ignore full destroy to speed up tests
+			"module.observability_agents_1.helm_release.logdna_agent",
+			"module.observability_agents_1.helm_release.sysdig_agent",
+			"module.observability_agents_2.helm_release.logdna_agent",
+			"module.observability_agents_2.helm_release.sysdig_agent",
+		},
+		// Do not hard fail the test if the implicit destroy steps fail to allow a full destroy of resource to occur
+		ImplicitRequired: false,
+		TerraformVars: map[string]interface{}{
+			"ocp_version": ocpVersion1,
+		},
+	})
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunSzClusterExample(t *testing.T) {
+	t.Parallel()
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  "examples/single_zone_cluster",
+		Prefix:        "sz-cluster",
+		ResourceGroup: resourceGroup,
+		ImplicitDestroy: []string{
+			"module.ocp_base.null_resource.confirm_network_healthy",
+			"module.ocp_base.null_resource.reset_api_key",
+		},
+		// Do not hard fail the test if the implicit destroy steps fail to allow a full destroy of resource to occur
+		ImplicitRequired: false,
+		TerraformVars: map[string]interface{}{
+			"ocp_version": ocpVersion3,
+		},
+	})
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunApplyTaintsExample(t *testing.T) {
+	t.Parallel()
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  "examples/apply_taints",
+		Prefix:        "std-taint",
+		ResourceGroup: resourceGroup,
+		ImplicitDestroy: []string{
+			"module.ocp_base.null_resource.confirm_network_healthy",
+			"module.ocp_base.null_resource.reset_api_key",
+		},
+		// Do not hard fail the test if the implicit destroy steps fail to allow a full destroy of resource to occur
+		ImplicitRequired: false,
+		TerraformVars: map[string]interface{}{
+			"ocp_version": ocpVersion4,
+		},
+	})
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
```

Comment:
Since none of these tests run in the PR pipeline, can you please attached proof that these tests are passing. Thanks

Reply:
Yes, sure. Here are the proofs for other three tests present in `other_tests.go` using three different ocp versions: 

| Example  | Results | Image (ocp version) |
| ------------- | ------------- |------------- |
| TestRunApplyTaints  | [TestRunApplyTaintsExample.log](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/files/10911681/TestRunApplyTaintsExample.log) | <img width="1224" alt="image" src="https://user-images.githubusercontent.com/78964991/223485645-5134175f-62fc-4ca3-ac7a-c5130a86d54e.png"> |
| TestRunMultiCluster | [TestRunMultiClusterExample.log](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/files/10911682/TestRunMultiClusterExample.log) | <img width="1421" alt="image" src="https://user-images.githubusercontent.com/78964991/223487457-44d33595-fd4b-4d1e-a5b2-aee2d4e4f8d7.png"> |
| TestRunSzCluster | [TestRunSzClusterExample.log](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/files/10911683/TestRunSzClusterExample.log) | <img width="1404" alt="image" src="https://user-images.githubusercontent.com/78964991/223487843-d90e0ded-16e4-4eb6-a688-d413988c7e8f.png"> |